### DISCUSSION TITLE
menu sliders "head" and "body" opacity overlapping

### DISCUSSION THREAD

**Discussion by integerrr** [ppy/osu#15144] [Q&A] [answered]
Not sure how to put this to words, when the menu sliders are half faded (unfocused?) the slider head's and the body's luma values seems to add together and forms a slightly brighter spot. 
![image](https://user-images.githubusercontent.com/7279624/137606700-49382806-497a-4464-abfd-ca8591c5f4b6.png)
![image](https://user-images.githubusercontent.com/7279624/137606702-cbad8f5a-1d04-47b9-a3f8-b2834b5f8baf.png)

![image](https://user-images.githubusercontent.com/7279624/137606694-47f61aa3-b1cf-4497-9a98-8b2fbef3a049.png)

**Comment by peppy** [comment, marked as answer]
99% sure this is already tracked elsewhere. we're aware of this and a fix is not super simple. Same thing happens on song select footer.

**Reply by integerrr** [reply]
#5923
